src/main/java/prefs/Preferences.java

```diff
@@ -15,83 +17,134 @@
+     * @return The initialised Preferences instance
+     */
+    public static Preferences load(String sessionConfigFileName, String userConfigFileName) {
+        return new Preferences(sessionConfigFileName, userConfigFileName, false);
     }
 
     /**
-     * Initialises a Preferences instance which creates its config file, or loads
-     * from it if it already exists.
+     * Initialises a Preferences instance that contain a clean slate of configuration values in the config files
+     * specified
+     * @param sessionConfigFileName The file name of the session config file which stores session config values.
+     * @param userConfigFileName The file name of the user config file which stores user config values.
+     * @return The initialised Preferences instance
      */
-    public static Preferences load(String configFileName) {
-        return new Preferences(configFileName, false);
+    public static Preferences create(String sessionConfigFileName, String userConfigFileName) {
+        return new Preferences(sessionConfigFileName, userConfigFileName, true);
     }
 
     /**
-     * Initialises a Preferences instance which always creates its config file.
-     * This will overwrite it with a default configuration if it already exists.
+     * @param sessionConfigFileName The name of the session config file
+     * @param ignoreExisting True if existing config file is to be ignored.
      */
-    public static Preferences create(String configFileName) {
-        return new Preferences(configFileName, true);
+    private Preferences(String sessionConfigFileName, String userConfigFileName, boolean ignoreExisting) {
+        this.sessionConfigFileName = sessionConfigFileName;
+        this.userConfigFileName = userConfigFileName;
+
+        this.sessionConfig = (SessionConfig) loadConfig(sessionConfigFileName, SessionConfig.class, ignoreExisting);
+        this.userConfig = (UserConfig) loadConfig(userConfigFileName, UserConfig.class, ignoreExisting);
     }
 
-    public void saveGlobalConfig() {
-        fileHandler.saveGlobalConfig(global);
+    /**
+     * Loads a Config instance from a file on disk
+     * @param configFileName The filename of the config file on disk
+     * @param configClass The class of the config to load
+     * @param ignoreExisting True if ignore the config file already
+     * @return The loaded Config instance
+     */
+    private Config loadConfig(String configFileName, Class<? extends Config> configClass, boolean ignoreExisting) {
+        FileHelper configFileHelper = new FileHelper(DIRECTORY, configFileName);
+        return new JsonHelper().createConfigFromJson(configFileHelper.loadFileContents(ignoreExisting), configClass);
     }
 
-    private void initGlobalConfig() {
-        global = fileHandler.initGlobalConfig();
+    /**
+     * Saves the session and user configs to file
+     */
+    public void flush() {
```

Comment:
The symmetric operation to `load` should probably be `save`. `flush` is a bit below the abstraction level of this class.